cartographer_rviz/cartographer_rviz/drawable_trajectory.cc

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2016 The Cartographer Authors
```

Comment:
Meh... I am really unsure how to correctly handle this. Maybe we can just remove the years in the copyright headers eventually. For now, just keep it 2016.